apps/climatemappedafrica/src/lib/data/common/index.js

```diff
@@ -74,12 +95,20 @@ export async function getPageProps(api, context) {
     page: { value: explorePage },
   } = hurumap;
 
-  const blocks = await blockify(page.blocks, api, context);
+  let blocks = await blockify(page.blocks, api, context);
   const variant = page.slug === explorePage.slug ? "explore" : "default";
 
   const siteSettings = await api.findGlobal("settings-site");
   const footer = getFooter(siteSettings, variant);
-  const menus = getNavBar(siteSettings, variant);
+  const menus = getNavBar(siteSettings, variant, explorePage);
+
+  if (slug === explorePage.slug) {
```

Comment:
I think we changed params from slug to `slugs` in line 80 above. We can fix it in this PR.